exporter/clickhouseexporter/internal/summary_metrics.go

```diff
@@ -156,7 +157,7 @@ func (s *summaryMetrics) insert(ctx context.Context, db *sql.DB) error {
 	return nil
 }
 
-func (s *summaryMetrics) Add(metrics any, metaData *MetricsMetaData, name string, description string, unit string) error {
+func (s *summaryMetrics) Add(resAttr map[string]string, resURL string, scopeInstr pcommon.InstrumentationScope, scopeURL string, metrics any, name string, description string, unit string) error {
```

Comment:
What about making the signature shorter this way?

```suggestion
func (s *summaryMetrics) Add(res pmetric.ResourceMetrics, scope pmetric.ScopeMetrics, metrics any, name string, description string, unit string) error {
```

Reply:
Please ignore this comment